run.sh

```diff
@@ -45,10 +43,12 @@ else
     
 fi 
 
+DOCKER_RUN_COMMAND="yum -y install git && ansible-galaxy install -r /tmp/src/requirements.yml --roles-path=/tmp/src/galaxy && $DOCKER_RUN_COMMAND"
```

Comment:
To install roles from git repository, git command have to installed in a container.

* https://github.com/rht-labs/labs-ci-cd/blob/master/galaxy_requirements.yml#L7
* https://github.com/rht-labs/labs-ci-cd/blob/master/requirements.yml#L5

Install git on the base image may be better...
https://github.com/redhat-cop/openshift-applier/blob/master/images/openshift-applier/Dockerfile